.cicdtemplate/.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -40,21 +43,21 @@ jobs:
       - name: Run Detekt
         run: ./gradlew detekt
 
-      - name: Archive Detekt report
+      - name: Archive Detekt reports
         uses: actions/upload-artifact@v2
         with:
-          name: detekt-report
+          name: detekt-reports
           path: build/reports/detekt/
 
-      - name: Run Unit Tests & Jacoco
+      - name: Run unit tests and Jacoco
         run: ./gradlew jacocoTestReport
 
-      # Archive Unit test & code coverage report
-      - name: Archive Code coverage report
+      # Archive Jacoco code coverage reports
+      - name: Archive code coverage reports
         uses: actions/upload-artifact@v2
         if: always()
         with:
-          name: code-coverage-report
+          name: code-coverage-reports
```

Comment:
not sure if this is a pure name or it's a path/params 🤔  if it's a pure name I think we shouldn't have `-` yeah?